systemtest/src/main/java/io/strimzi/kafka/access/installation/BundleInstallation.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.access.installation;
+
+import io.fabric8.kubernetes.api.model.NamespaceBuilder;
+import io.fabric8.kubernetes.api.model.ServiceAccount;
+import io.fabric8.kubernetes.api.model.ServiceAccountBuilder;
+import io.fabric8.kubernetes.api.model.apiextensions.v1.CustomResourceDefinition;
+import io.fabric8.kubernetes.api.model.apps.Deployment;
+import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRole;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRoleBinding;
+import io.fabric8.kubernetes.api.model.rbac.ClusterRoleBindingBuilder;
+import io.skodjob.testframe.installation.InstallationMethod;
+import io.skodjob.testframe.resources.KubeResourceManager;
+import io.skodjob.testframe.utils.ImageUtils;
+import io.skodjob.testframe.utils.TestFrameUtils;
+import io.strimzi.kafka.access.Environment;
+import io.strimzi.kafka.access.TestConstants;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+
+public class BundleInstallation implements InstallationMethod {
+    private final String installationNamespace;
+
+    public BundleInstallation(String installationNamespace) {
+        this.installationNamespace = installationNamespace;
+    }
+
+    @Override
+    public void install() {
+        List<File> accessOperatorFiles = Arrays.stream(new File(TestConstants.INSTALL_PATH).listFiles())
+            .sorted()
+            .filter(File::isFile)
+            .toList();
+
+        accessOperatorFiles.forEach(file -> {
+            final String resourceType = file.getName().split("-")[1].replace(".yaml", "");
```

Comment:
This is good place for future errors I think, but atm I don't have much better solution in mind. It should be fine until we will run e2e tests for each change that will include changes in the file names.

Reply:
That's how it is done in Strimzi as well and I don't think that there are a lot of changes to the file names TBH.
But I can think about better solution

Reply:
TBH not sure if there is a better way...I was just thinking loudly :) I will resolve this and in case we will find a better way we can discuss it.

Reply:
Sure thing, thanks :) and good catch.